src/Behat/Behat/Output/Node/Printer/Progress/ProgressStepPrinter.php

```diff
@@ -52,6 +52,17 @@ public function printStep(Formatter $formatter, Scenario $scenario, StepNode $st
         $printer = $formatter->getOutputPrinter();
         $style = $this->resultConverter->convertResultToString($result);
 
+        // FIXME when refactored to symphony output use OutputInterface->getVerbosity() here instead of the magic global
+        $isVeryVerbose = $_SERVER['SHELL_VERBOSITY'] ?? 0 >= 2;
```

Comment:
im very aware that this looks hacky, but i thought it would increase the odds of this being merged as bugfix?

Anyway i was not able to get the symphony output instance here as it seems behat is using an completely custom outputPrinter (probably due to history).

So i assume that once symphony is used here we can cleanly access the verbosity.

On the other hand, i think we could even drop the verbosity thing here and always dump the stdout as this is helpful for debugging, and in CI there should ideally be no output in general.

One more thing: The behat output has a verbosity option via `$printer->getOutputVerbosity()` which can only be set via: `--format-settings '{"output_verbosity":2}'` (it will not respect the global app option `-vv`)
But `getOutputVerbosity` is declared as `@deprecated since 3.1, to be removed in 4.0` so i dindt want to use that.